nodejs/awsx/package.json

```diff
@@ -12,7 +12,7 @@
     "repository": "https://github.com/pulumi/pulumi-awsx",
     "dependencies": {
         "@pulumi/pulumi": "^0.17.1",
-        "@pulumi/aws": "^0.17.0",
+        "@pulumi/aws": "0.17.2-dev.1553126336",
```

Comment:
is this appropriate?  why can't we just reference the latest aws?  or will that come once you publishthe next aws version?

Reply:
This (for now) was to reference a known version, but I'll change it to a proper version constraint once we start releasing these.